test/bdd/fixtures/scripts/strapi_configure.sh

```diff
@@ -37,7 +37,7 @@ do
    then
      break
    fi
-   n=$[$n+1]
```

Comment:
This is an unrelated bug I fixed. This wasn't actually incrementing the value of `n`, instead it was literally appending it a bunch of times like this: `$[$[$[0+1]+1]+1]`